idanalysis/idsdata.py

```diff
@@ -565,6 +566,138 @@ def get_fmap_fname(self, phase):
         return fmap_fname
 
 
+class APU22MANACAData(APU22Data):
+    PARAMS = _PARAMS()
+    PARAMS.B_PEAK = 0.7  # [T]
+    PARAMS.PERIOD_LEN = 22  # [mm]
+    PARAMS.ID_LEN = 1.300  # [m]
+    PARAMS.NR_PERIODS = 51
+    PARAMS.KPARAMETER_NAME = "phase"
+    PARAMS.ID_FAMNAME = "APU22"
+    PARAMS.SUBSECS = ["ID09SA"]
+
+    PARAMS.KICKMAPS_DATA_PATH = REPOS_PATH + "apu22/kickmaps/"
+    PARAMS.FIELDMAPS_DATA_PATH = (
+        DATA_REPOS_PATH
+        + "lnls-ima/kyma22/id-manaca/commissioning_id/measurement/magnetic/"
+        + "lnls/hallprobe/vertical_position_0mm/"
+    )  # noqa: E501
+    PARAMS.FOLDER_BASE_OUTPUT = REPOS_PATH + "apu22/results/data/"
+
+    FIELMAPS_CONFIGS = {
+        "ID2828": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2828_Phase=0mm.dat",
+        "ID2829": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2829_Phase=1mm.dat",
+        "ID2830": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2830_Phase=2mm.dat",
+        "ID2831": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2831_Phase=3mm.dat",
+        "ID2832": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2832_Phase=4mm.dat",
+        "ID2833": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2833_Phase=5mm.dat",
+        "ID2834": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2834_Phase=6mm.dat",
+        "ID2835": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2835_Phase=7mm.dat",
+        "ID2836": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2836_Phase=8mm.dat",
+        "ID2837": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2837_Phase=9mm.dat",
+        "ID2838": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2838_Phase=10mm.dat",
+        "ID2839": "2020-05-26_1991b_Fieldmap_X=-12_12mm_Z=-740_740mm_"
+        + "Y=0mm_ID=2839_Phase=11mm.dat",
+    }
+
+    phase_dict = {
```

Comment:
you don't need the parenthesis on the key definitions.

Why this needs to be a dictionary? A simple list wouldn't suffice?
Considering it is a class constant, its name should be in capital letters.

Reply:
I use parenthesis because we need a tuple here. Sometimes, we have a phase_gap_dict, for example. So, we need to map the dictionary keys (tuples) into their file code.